lib/properties.js

```diff
@@ -29,9 +32,8 @@ class Properties extends EventEmitter {
       // Get Dynamic layers' source instances
       .concat(this.layers.map((layer) => layer.source).filter((source) => !!source))
 
-      // Add the active View's sources. Reversed for precedence consistency with Layers:
-      // Entry 0 is most precedent, LAST is least precedent.
-      .concat(this.active.sources.reverse());
+      // Add the active View's sources.
+      .concat(this.active.sources);
```

Comment:
I feel like this is diminishing returns. The only use of this getter is in the health check and the order doesn't change anything beyond which order the source statuses are displayed in the `/v1/status` output.